packages/nextjs/services/web3/wagmi-burner/BurnerConnector.ts

```diff
@@ -78,15 +80,15 @@ export class BurnerConnector extends Connector<StaticJsonRpcProvider, BurnerConn
     return Promise.resolve();
   }
 
-  async getAccount(): Promise<string> {
+  async getAccount(): Promise<`0x${string}`> {
```

Comment:
it seems it can be undefined too

Reply:
Thanks, handled this at [54999ee](https://github.com/scaffold-eth/se-2/pull/96/commits/54999ee7e063e97f1cb5be66c58f6c0d6d595a44)